x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/plan/logical/command/sys/SysTypesTests.java

```diff
@@ -132,6 +139,33 @@ public void testSysTypes() {
         }, ex -> fail(ex.getMessage())));
     }
 
+    public void testUnsignedLongFiltering() {
+        Set<SqlVersion> versions = new HashSet<>(
+            List.of(
+                SqlVersion.fromId(INTRODUCING_UNSIGNED_LONG.id - SqlVersion.MINOR_MULTIPLIER),
+                INTRODUCING_UNSIGNED_LONG,
+                SqlVersion.fromId(INTRODUCING_UNSIGNED_LONG.id + SqlVersion.MINOR_MULTIPLIER),
+                SqlVersion.fromId(Version.CURRENT.id)
```

Comment:
Same here: this list can be extracted in a common place and re-used.

Reply:
Suggestion applied.